Maddori.Apple/Maddori.Apple/Global/UIComponent/View/EmptyView/EmptyFeedbackView.swift

```diff
@@ -9,7 +9,47 @@ import UIKit
 
 import SnapKit
 
-final class EmptyFeedbackView: BaseCollectionViewCell {
+final class EmptyCollectionFeedbackView: BaseCollectionViewCell {
+    
+    private enum Size {
+        static let capsuleWidth: CGFloat = 88
+        static let capsuleHeight: CGFloat = 54
+        static let capsuleYOffset: CGFloat = 20
+        static let labelYOffset: CGFloat = 25
+    }
+    
+    // MARK: - property
+    
+    private let emptyFeedbackKeyword = EmptyFeedbackKeyword()
+    lazy var emptyFeedbackLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
+        label.textAlignment = .center
+        label.font = .body3
+        label.textColor = .gray700
+        return label
+    }()
+    
+    // MARK: - life cycle
+    
+    override func render() {
```

Comment:
p1;

```suggestion
    // MARK: - life cycle
    
    override func configUI() {
        backgroundColor = .backgroundWhite
    }
```

요게 추가되어야 합니다 !
'매'의 눈으로 보면 색상이 차이나는 걸 볼 수 있습니다 !
<img width="394" alt="image" src="https://user-images.githubusercontent.com/78677571/202903747-b8ef61d5-57e5-4dfb-9359-fe20c19895a2.png">

Reply:
진짜 어케봤어요....
[61c06b4](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/146/commits/61c06b49bd225258e5bf26a62c3a38eb3be4938e)